src/notation/qml/MuseScore/NotationScene/internal/DynamicPopup.qml

```diff
@@ -51,10 +51,40 @@ StyledPopupView {
         NavigationPanel {
             id: dynamicsNavPanel
             name: "DynamicsPopup"
-            direction: NavigationPanel.Vertical
+            direction: NavigationPanel.Horizontal
             section: root.notationViewNavigationSection
             order: root.navigationOrderStart
             accessible.name: qsTrc("notation", "Dynamics Popup")
+
+            onNavigationEvent: function(event) {
+                if (event.type === NavigationEvent.Escape) {
+                    root.close()
```

Comment:
Do we know why the `closeOnEscape` property of the `StyledPopupView` is not working for these popups? It's on by default, and the 'close on escape' behaviour already seems to be implemented in `PopupContent`.

Reply:
Because we don't use the popup navigation section, but use the notation navigation section. This is done because we don't need to focus to the popup when it opens